bin/translate_images.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2023 Amazon.com, Inc. or its affiliates.
+
+"""
+###################################################################################################
+
+This Python script provides functionality to convert GeoTIFF images to NITF (National Imagery Transmission Format)
+format. The script supports two target NITF formats: greyscale and WIBD (Wavelet Image-Based Data). It uses the GDAL
```

Comment:
Is it Wavelet Image-Based Data or Wideband Image Data

Reply:
Updated